### PR TITLE
Signup page update

### PR DESCRIPTION
**Pull request by ayushi-siddhu**
image
image

### CODE REVIEW

File: public/css/neumorphism.css [hunk 1]
```diff
@@ -32278,4 +32278,29 @@ article a:hover {
     box-shadow: inset 2px 2px 5px #b8b9be, inset -3px -3px 7px #FFFFFF;
 }
 
+.login {
```
Comment (chayan007) [inline public/css/neumorphism.css:32281]: As discussed, move this personalized css into component's scoped `<style>`

File: public/css/neumorphism.css [hunk 2]
```diff
@@ -2982,7 +2982,7 @@ fieldset:disabled a.btn {
 }
```
Comment (chayan007) [inline public/css/neumorphism.css:2983]: Any changes made in this global css, needs to be overwritten in the component's scoped `<style>`

File: src/components/SignUp.vue [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<template>
+  <section class="section section bg-soft pb-5 overflow-hidden z-2">
+            <div class="container z-2">
+            <SplashScreen></SplashScreen>
```
Comment (chayan007) [inline src/components/SignUp.vue:4]: Why is the `<SplashScreen/>` used in signup component ?

Comment (chayan007) [reply src/components/SignUp.vue:4]: @itisdb please check this.

**Review by chayan007 — CHANGES_REQUESTED** [review]

File: src/components/SignUp.vue (hunk 3 above)
Comment (itisdb) [reply src/components/SignUp.vue:4]: The splash screen component had image in it as well. That's the reason!

File: public/css/neumorphism.css (hunk 2 above)
Comment (chayan007) [reply public/css/neumorphism.css:2983]: @itisdb please help her in this change.

File: src/components/SignUp.vue (hunk 3 above)
Comment (chayan007) [reply src/components/SignUp.vue:4]: Oh LOL ! Can we use the image directly here ?

**Comment by vikas339** [comment]
it looks good.
I think the text "express yourself" should small.
That is my job, should I do that?

**Comment by chayan007** [comment]
> it looks good.
> I think the text "express yourself" should small.
> That is my job, should I do that?

@vikas339 guide her to the change. Do not commit in someone else's PR. Or what we can do is, let her merge this PR (after getting validated from @itisdb) and then you can pull it in and change accordingly.

File: public/css/neumorphism.css (hunk 1 above)
Comment (ayushi-siddhu) [reply public/css/neumorphism.css:32281]: okay

File: src/components/SignUp.vue (hunk 3 above)
Comment (itisdb) [reply src/components/SignUp.vue]: image
image


The Code has been updated.

**Review by chayan007 — APPROVED** [review]

**Review by chayan007 — APPROVED** [review]

File: src/App.vue [hunk 4]
```diff
@@ -1,12 +1,16 @@
 <template>
 <nav id="navbar-main" aria-label="Primary navigation" class="navbar navbar-main navbar-expand-lg navbar-theme-primary headroom navbar-light navbar-transparent headroom--not-top headroom--not-bottom headroom--pinned">
    <div class="d-flex align-items-center">
-    <button type="submit" class="btn btn-primary ml-2"> <router-link to="/">Home</router-link></button>
+    <button type="submit" class="btn btn-primary ml-2"> <router-link to="/signup">SignUp</router-link></button>
     <button type="submit" class="btn btn-primary ml-2"> <router-link to="/register">Register</router-link></button>
     <button type="submit" class="btn btn-primary ml-2"> <router-link to="/splashscreen">SplashScreen</router-link></button>
+    <button type="submit" class="btn btn-primary ml-2"> <router-link to="/baselayout">Baselayout</router-link></button>
```
Comment (chayan007) [inline src/App.vue:7]: Why is BaseLayout invoked here ?

**Review by chayan007 — CHANGES_REQUESTED** [review]

**Review by itisdb — APPROVED** [review]

**Review by chayan007 — APPROVED** [review]

**Pull request merged**